### ISSUE TITLE
Carousel with Pagination has no focus outline on the previous and next buttons

### ISSUE THREAD

**Issue by kolaps33** [microsoft/fluentui#14027]
Issue reported by user testing.

Environment Information
Win10
Chrome Version 83.0.4103.116
NVDA 2020.1

### Describtion:
The Previous / next buttons in ‘Carousel with pagination’ are not highlighted when they get the focus. 

#### Repro steps:
1. open page https://fluentsite.z22.web.core.windows.net/components/carousel/definition
2.  navigate into the example https://fluentsite.z22.web.core.windows.net/components/carousel/definition
3. navigate to previous or to the next button
4. observe focus outline for the button

#### Actual behavior:
there should be focus outline

#### Expected behavior:
there is no focus outline
